server/app/views/api/ApiDocsView.java

```diff
@@ -83,31 +84,50 @@ public Content render(
         layout
             .getBundle(request)
             .setTitle("API Docs")
-            .addMainContent(contentDiv(programDefinition, allProgramSlugs, request))
+            .addMainContent(
+                contentDiv(selectedProgramSlug, programDefinition, allProgramSlugs, request))
             .addMainStyles("overflow-hidden");
 
     return layout.render(bundle);
   }
 
   private DivTag contentDiv(
-      ProgramDefinition programDefinition,
+      String selectedProgramSlug,
+      Optional<ProgramDefinition> programDefinition,
       ImmutableSet<String> allProgramSlugs,
       Http.Request request) {
 
     SelectTag slugsDropdown =
         select()
             .withId("select-slug")
-            .withClasses("border", "border-gray-300", "rounded-lg", "p-2", "ml-2")
-            .attr("onchange", "window.location.href = '/api/docs/v1/' + this.value");
-
+            .withClasses("border", "border-gray-300", "rounded-lg", "p-2", "mx-2")
+            .attr("onchange", "window.location.href = '/api/docs/v1/' + this.value + '/active'");
```

Comment:
**suggestion (blocking):** please move this code into a typescript file and follow the pattern used for registering DOM event handlers used on other pages

Reply:
Done.